lib/__tests__/_cookieUtils.test.ts

```diff
@@ -63,6 +65,9 @@ describe("cookieUtils", () => {
     });
   });
   describe("getUserToken", () => {
+    beforeEach(() => {
+      AlgoliaInsights.setUserToken("007");
+    });
     it("should return the current userToken", () => {
```

Comment:
This test case was relying on the previous cases' output.
Now that we freshly create an instance, I had to set value before the tests to make it work correctly.